test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -353,56 +353,6 @@ def test_text_measure_2_reverse_bits(self):
             expected,
         )
 
-    def test_wire_order(self):
-        """Test the wire_order option"""
-        expected = "\n".join(
-            [
-                "                  ",
-                "q_2: |0>──────────",
-                "        ┌───┐     ",
-                "q_1: |0>┤ X ├─────",
-                "        ├───┤┌───┐",
-                "q_3: |0>┤ H ├┤ X ├",
-                "        ├───┤└─╥─┘",
-                "q_0: |0>┤ H ├──╫──",
-                "        └───┘  ║  ",
-                " c_2: 0 ═══════o══",
-                "               ║  ",
-                "ca_0: 0 ═══════╬══",
-                "               ║  ",
-                "ca_1: 0 ═══════╬══",
-                "               ║  ",
-                " c_1: 0 ═══════■══",
-                "               ║  ",
-                " c_0: 0 ═══════o══",
-                "               ║  ",
-                " c_3: 0 ═══════■══",
-                "              0xa ",
-            ]
-        )
-
-        qr = QuantumRegister(4, "q")
-        cr = ClassicalRegister(4, "c")
-        cr2 = ClassicalRegister(2, "ca")
-        circuit = QuantumCircuit(qr, cr, cr2)
-        circuit.h(0)
-        circuit.h(3)
-        circuit.x(1)
-        with self.assertWarns(DeprecationWarning):
-            circuit.x(3).c_if(cr, 10)
-        self.assertEqual(
-            str(
-                circuit_drawer(
-                    circuit,
-                    output="text",
-                    initial_state=True,
-                    cregbundle=False,
-                    wire_order=[2, 1, 3, 0, 6, 8, 9, 5, 4, 7],
-                )
-            ),
-            expected,
-        )
```

Comment:
This test looks still correct, just we need to use something else for the conditional bits (`if_test`?)

Reply:
Restored in: https://github.com/Qiskit/qiskit/pull/13506/commits/6f3861acbf80a115679ffa7f66f55a108b3c6af8